src/aura/CMP_Acc_Naming_Dropdown/CMP_Acc_Naming_DropdownHelper.js

```diff
@@ -0,0 +1,82 @@
+({
+    init : function(component) {
+        //Retrieving account naming options
+        this.getAdminAccNameFormatOptions(component);
+        this.getHHAccNameFormatOptions(component);
+    },
+    onSelectChange : function(component) {
+        var selectedVal = component.find("nameFormatDropDown").get("v.value");
+        if (prefix === 'hed__') {
```

Comment:
Looks like the prefix variable is not defined here. So, the HEDA setting page does not load.